app/models/manageiq/providers/ibm_cloud/vpc/manager_mixin.rb

```diff
@@ -16,7 +16,11 @@ def connect(options = {})
     key = authentication_key(options[:auth_type])
     region = options[:provider_region] || provider_region
     sdk = self.class.raw_connect(key)
-    sdk.vpc(:region => region)
+    if options[:service] == 'ActivityTrackerApiV1'
+      sdk.events(:region => region)
```

Comment:
So this is where we're going to want to pull in the service key since we have the proper provider context, rather than https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/248/files#diff-ecb90bb1257297ad164960093c29c4d725223dbcfd4a9bd36cbbb6f2f30a5ac4R17 which would be the same for all provider instances.

You can do `service_key = authentication_key("events")` here